README.md

```diff
@@ -125,55 +125,46 @@ podman machine start
 
 ## Installation
 
-This extension can be installed through the **Extensions** page of Podman Desktop:
+This extension can be installed through the **Extensions** section of Podman Desktop:
 
-1. Go to *Settings > Extensions*.
-2. Copy and paste the below image to the *Name of the image* field:
+1. Go to *Extensions* in the navbar.
+2. Copy and paste the below image to the *Install custom...* section:
```

Comment:
We should direct users to the Catalog instead. By early next week we would be listed there, and easier than copy/paste.

Reply:
I agree, should we update this once it's in the catalog?

Reply:
We have a chicken and egg issue here: if we commit a change now then it'll be wrong for anyone who tries to install over the next week; if we don't then the copy in the catalog would contain text on installing via custom link. Unless we patch the catalog next week to not match what was in the release, which is also kind of ugly...

Since there will be far more people looking at this next week, I think the simplest is to just update it now. If anyone can't figure out how to install in the meantime hopefully they'll ping us.

Reply:
added https://github.com/containers/podman-desktop-extension-bootc/issues/413